engine/render/src/render/render_script.cpp

```diff
@@ -1025,9 +1020,8 @@ namespace dmRender
                 }
                 else
                 {
-                    lua_pop(L, 2);
```

Comment:
This is the error. We only popped two values when the stack contained four values.